src/target/riscv/riscv.c

```diff
@@ -4285,6 +4354,20 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.usage = "address value",
 		.help = "Perform a 32-bit DMI write of value at address."
 	},
+	{
+		.name = "dm_read",
+		.handler = riscv_dm_read,
+		.mode = COMMAND_ANY,
+		.usage = "address",
+		.help = "Perform a 32-bit DM read at address, returning the value."
```

Comment:
One more detail - relates to the already-changed documentation:

```suggestion
		.handler = riscv_dm_read,
		.mode = COMMAND_ANY,
		.usage = "reg_address",
		.help = "Perform a 32-bit read from DM register at reg_address, returning the value."
```